### PR TITLE
Update Portuguese translation

### PR DESCRIPTION
**Pull request by fabio-ricardo**

### CODE REVIEW

File: src/lib/assets/i18n/pt.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
     "description": "Esta ferramenta foi originalmente desenhada pelo <a target=\"_blank\" href=\"https://www.cnil.fr\">Comissão Nacional da Informática e das Liberdades (CNIL)</a>, Autoridade Francesa de Proteção de Dados, para ajudar o subcontratante a ser compatível com o GDPR. It was developed by <a href=\"https://atnos.com\" target=\"_blank\">ATNOS</a>.<br>Este software é gratuito e de código aberto. Caso pretenda se envolver neste projeto, poderá baixar o código fonte do Github, modificá-lo e compartilhá-lo com a comunidade.",
     "licences": "Licensas das bibliotecas:",
-    "title": "Sobre a Ferramenta PIA",
-    "translators": "O software PIA está atualmente disponível em dezoito idiomas. As versões francesa e inglesa são fornecidas pelo CNIL. As outras versões resultam das contribuições dos seguintes indivíduos:",
+    "title": "Sobre a Ferramenta AIPD",
+    "translators": "O software AIPD está atualmente disponível em dezoito idiomas. As versões francesa e inglesa são fornecidas pelo CNIL. As outras versões resultam das contribuições dos seguintes indivíduos:",
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:7]: This should keep PIA. We are talking about the software PIA here not the AIPD.
See: https://github.com/LINCnil/pia-i18n/blob/7a13930809db70b8867d279f5cda5977e2370892/src/lib/assets/i18n/en.json#L6-L7
Thx

File: src/lib/assets/i18n/pt.json [hunk 2]
```diff
@@ -23,81 +23,81 @@
...
+    "sign_in": "Login",
+    "sign_in_button": "Login",
+    "sign_out_button": "Sair",
+    "special_character": "Caracter especial",
     "start": "Início",
-    "start_app": "Aceder ao software PIA (Beta)",
-    "subtitle1": "Uma plataforma para criar <br> e gerir as análises de PIA",
-    "subtitle2": "Aceder a ferramentas e glossários",
+    "start_app": "Acessar ao software AIPD (Beta)",
+    "subtitle1": "Uma plataforma para criar <br> e gerir as análises de AIPD",
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:53]: Same as above

File: src/lib/assets/i18n/pt.json [hunk 3]
```diff
@@ -167,13 +167,13 @@
     "to_correct": "A corrigir"
   },
   "header": {
-    "archives": "TO TRANSLATE - PIA Archives",
-    "current_pias": "TO TRANSLATE - Current PIAs",
-    "exit_full_screen_mode": "Pressione F11 para sair do mode de ecrã completo.",
+    "archives": "Arquivos AIPD",
+    "current_pias": "AIPDs atuais",
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:171]: Same as above

File: src/lib/assets/i18n/pt.json [hunk 4]
```diff
@@ -167,13 +167,13 @@
...
+    "exit_full_screen_mode": "Pressione F11 para sair do modo tela cheia.",
     "help": {
-      "issue_reporting": "TO TRANSLATE - Report an issue",
-      "manual": "TO TRANSLATE - User manual",
-      "pia_example_link": "Exemplo PIA",
+      "issue_reporting": "Reportar um problema",
+      "manual": "Manual do usuário",
+      "pia_example_link": "Exemplo AIPD",
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:176]: Same as above

File: src/lib/assets/i18n/pt.json [hunk 5]
```diff
@@ -207,52 +207,52 @@
       "sl": "Slovenščina",
       "sv": "Svenska"
     },
-    "logo_subtitle": "privacy impact assessment",
+    "logo_subtitle": "avaliação de impacto da proteção de dados",
     "logo_title": "Avaliação de Impacto da Proteção de Dados",
-    "structures": "Modelos de PIA",
+    "structures": "Modelos de AIPD",
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:212]: Same as above

File: src/lib/assets/i18n/pt.json [hunk 6]
```diff
@@ -207,52 +207,52 @@
...
+      "url_setting": "Configurar URL do servidor",
+      "users": "Usuários"
     }
   },
   "help": {
     "definition": {
       "data1": {
         "content": "Contéudo sendo escrito",
-        "title": "Usa a ferramente PIA"
+        "title": "Use a ferramente AIPD"
```
Comment (brunto) [inline src/lib/assets/i18n/pt.json:226]: Same as above

**Review by brunto — CHANGES_REQUESTED** [review]
Thx for your translations. I added some feedback.
Globally you should not change PIA to AIPD. Could you revert this everywhere?

**Comment by fabio-ricardo** [comment]
Thanks for the feedback! Changes have been performed accordingly. Please let me know in case you spot anything else.

**Review by brunto — APPROVED** [review]

**Comment by brunto** [comment]
Thx a lot @fabio-ricardo

**Pull request merged**